library/ssl_srv.c

```diff
@@ -2801,9 +2801,9 @@ static void ssl_handle_id_based_session_resumption( mbedtls_ssl_context *ssl )
     }
 
     /* Move semantics */
-    /* Zeroization of session_tmp happens at the end of the function. */
     mbedtls_ssl_session_free( session );
     *session = session_tmp;
+    memset( &session_tmp, 0, sizeof( mbedtls_ssl_session ) );
```

Comment:
Minor: Would be slightly more robust to mistakes later when the code changes (eg. if the type of `session_tmp` changes) like this:

```suggestion
    memset( &session_tmp, 0, sizeof( session_tmp ) );
```

Reply:
Changed.